data/json/mapgen/lab/lab_surface/lab_surface_big_z4.json

```diff
@@ -0,0 +1,246 @@
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 48, 57 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 59, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 84, 93 ], "repeat": 7 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 12, "y": 36 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 42 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 84 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 36 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 84 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 32, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 38, "y": 96 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 103, "y": 36 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 83, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 77, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 97, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 103, "y": 42 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 77, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 77, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 77, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 77, "y": 84 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 97, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 97, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 97, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 97, "y": 84 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 77, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 71, "y": 96 },
```

Comment:
I think the `x` on this is supposed to be 83. The current value of 71 spawns it over the skylight, and the room further east in the json has nothing in it.

![image](https://user-images.githubusercontent.com/11464/54487308-9ddf9980-4849-11e9-98c1-d152a6197597.png)